tb-dev-challenge/tb-dev-challenge/ItemCollectionViewController.swift

```diff
@@ -0,0 +1,124 @@
+//
+//  ItemCollectionViewController.swift
+//  tb-dev-challenge
+//
+//  Created by Bhavin Kapadia on 2022-02-27.
+//
+
+import UIKit
+import CoreData
+
+class ItemCollectionViewController: UICollectionViewController {
+
+    var menuGroupItems: [MenuItems]?
+    var menuGroup: MenuGroups?
+
+    let context = (UIApplication.shared.delegate as! AppDelegate).persistentContainer.viewContext
+    //    var menuGroup = MenuGroupElement(menuGroupName: "", menuGroupImage: "", menuGroupSet: false, menuItems: [])
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        title = menuGroup?.menuName
+        if let allMenuItems = menuGroup?.ofMenuItem?.allObjects as? [MenuItems] {
+            menuGroupItems = allMenuItems
+        }
+        self.navigationItem.rightBarButtonItem = self.editButtonItem
+        self.collectionView.reloadData()
+        fetchMenuGroupItems()
```

Comment:
if these 2 lines are called in `viewWillAppear` anyway, we don't need to call them here

Reply:
yea that's true - removed

Reply:
fixed in https://github.com/bkapadia01/HundredDaysSwift/pull/27/commits/a167ca8e3168185c4a3862128e81b076a3a95dad